de.dlr.sc.overtarget.language/src/de/dlr/sc/overtarget/language/util/TargetFileHandler.xtend

```diff
@@ -0,0 +1,61 @@
+/*******************************************************************************
+ * Copyright (c) 2020 German Aerospace Center (DLR), Simulation and Software Technology, Germany.
+ *
+ * This program and the accompanying materials are made available under the
+ * terms of the Eclipse Public License 2.0 which is available at
+ * http://www.eclipse.org/legal/epl-2.0.
+ *
+ * SPDX-License-Identifier: EPL-2.0
+ *******************************************************************************/
+package de.dlr.sc.overtarget.language.util
+
+import org.eclipse.core.resources.IFile
+import de.dlr.sc.overtarget.language.generator.OvertargetGenerator
+import org.eclipse.emf.common.util.URI
+import org.eclipse.emf.ecore.resource.ResourceSet
+import org.eclipse.emf.ecore.resource.impl.ResourceSetImpl
+import de.dlr.sc.overtarget.language.targetmodel.TargetFile
+
+class TargetFileHandler {
+	
+	def getTargetFile(IFile file, ResourceSet rs) {
```

Comment:
Please add some java doc comment

Reply:
And should it be getTargetResource ? As you are basically transforming an IFile to an EMF Resource